lib/token.h

```diff
@@ -811,10 +811,10 @@ class CPPCHECKLIB Token {
         }
     }
 
-    MathLib::bigint exprId() const {
+    nonneg int exprId() const {
```

Comment:
exprids can be a fraction of the total number of tokens. This probably wont be a problem unless we have preprocessed source files larger then 4G(which seems unlikely).

Reply:
yes it sounds unbelievable. I also have the feeling that analysis time in general more or less grows exponentially from the filesize.. so it would not be practical to analyze it anyway.